cypress/fixtures/settings.json

```diff
@@ -1,7 +1,7 @@
 [
   {
       "accountStatus": "ACTIVE",
```

Comment:
Can you change this accountType back to 'PREMIUM' so we don't always use a staff account?

Reply:
good catch done